less/pages/clubs.less

```diff
@@ -0,0 +1,74 @@
+.page.clubs main ul.clubs li {
+  margin: 0;
+  border: none;
```

Comment:
We need this super-specific selector so we can override the margin and border for all `li` elements, which I don't like. We should probably change the styling for the page so that `li` elements don't have orange borders and big margins by default.